core/v2/types.go

```diff
@@ -0,0 +1,96 @@
+package corev2
+
+import (
+	"math"
+
+	"github.com/Layr-Labs/eigenda/encoding"
+)
+
+var (
+	ParametersMap = map[uint8]BlobVersionParameters{
+		0: {CodingRate: 8, ReconstructionThreshold: 0.22, NumChunks: 8192},
+	}
+)
+
+type QuorumID = uint8
+
+type OperatorID = [32]byte
+
+// Assignment contains information about the set of chunks that a specific node will receive
+type Assignment struct {
+	StartIndex uint32
+	NumChunks  uint32
+}
+
+// GetIndices generates the list of ChunkIndices associated with a given assignment
+func (c *Assignment) GetIndices() []uint32 {
+	indices := make([]uint32, c.NumChunks)
+	for ind := range indices {
+		indices[ind] = c.StartIndex + uint32(ind)
+	}
+	return indices
+}
+
+// BlobHeader contains all metadata related to a blob including commitments and parameters for encoding
+type BlobHeader struct {
+	Version uint8
+
+	encoding.BlobCommitments
+
+	// QuorumInfos contains the quorum specific parameters for the blob
+	QuorumNumbers []uint8
```

Comment:
we should probably use something like `BlobQuorumInfo`

Reply:
Why do you say this? The v2 `BlobHeader` only needs `QuorumNumbers`

Reply:
Don't we need to know the security parameters in those quorums?

Reply:
This is derived from the blob version.

Reply:
I guess it's related to [this question](https://github.com/Layr-Labs/eigenda/pull/806#discussion_r1804103114). Should we get rid of the onchain parameters then?